### PR TITLE
UI updates & account functionality

### PR DESCRIPTION
**Pull request by benbrott**

### CODE REVIEW

File: SpareMe/screens/CreateAccount.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    onRegister = () => {
+        const { email, password } = this.state;
+        firebase.auth().createUserWithEmailAndPassword(email, password)
+        .then((user) => {
+          this.props.navigation.goBack();
+            // If you need to do anything with the user, do it here
+            // The user will be logged in automatically by the
+            // `onAuthStateChanged` listener we set up in App.js earlier
+        }).catch((error) => {
+            const { code, message } = error;
```
Comment (pg8wood) [inline SpareMe/screens/CreateAccount.js]: Does `message` give a reason for failure? It'd be cool if we could say "username already taken", "password doesn't meet requirements", etc

File: SpareMe/screens/Home.js [hunk 2]
```diff
@@ -86,6 +115,29 @@ export default class Home extends Component {
         this.setState({url: text});
     }
 
+    menuHandler = (value) => {
+        switch(value) {
+          case 1: //Sign In
```
Comment (pg8wood) [inline SpareMe/screens/Home.js]: You should use constants both here and in the `MenuOption` markup. Something akin to `constants.SIGN_IN`, etc. This makes your code much clearer

File: SpareMe/components/URLBar.js [hunk 3]
```diff
@@ -53,57 +55,70 @@ class URLBar extends Component {
...
+                  }>
+                      <Image source={require('./refresh.png')} style={styles.refresh}/>
+                  </TouchableOpacity>
+
+
+                  <Menu onSelect = {value => menuHandler(value)}>
+                    <MenuTrigger>
+                      <Image source={require('./menu.png')} style={styles.refresh}/>
+                    </MenuTrigger>
+                    <MenuOptions>
```
Comment (pg8wood) [inline SpareMe/components/URLBar.js]: These work well, but they're really difficult to tap. @wtravis1 can the menu options be made bigger and can the size of the menu be constrained to the length of the longest element?

File: SpareMe/screens/Home.js (hunk 2 above)
Comment (benbrott) [reply SpareMe/screens/Home.js]: Done

**Pull request merged**